.github/workflows/fedora-38.yml

```diff
@@ -552,3 +552,108 @@ jobs:
+    env:
+      STATUS_NAME: Tests / iot-simplified-f38
+
+    steps:
+      - name: Get Current Job Log URL
+        id: job-log-url
+        run: |
+          JOB_HTML_URL=$(curl --get -Ss -H "Authorization: token ${GITHUB_TOKEN}" -H "Accept: application/vnd.github.v3+json" "https://api.github.com/repos/osbuild/rhel-edge-ci/actions/runs/${GITHUB_RUN_ID}/jobs?per_page=30" | jq -r --arg job_name "pre-iot-simplified-f38" '.jobs | map(select(.name == "pre-iot-simplified-f38")) | .[0].html_url')
+          echo "html_url=$JOB_HTML_URL" >> $GITHUB_OUTPUT
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Create in-progress status
+        uses: octokit/request-action@v2.x
+        with:
+          route: 'POST /repos/osbuild/osbuild-composer/statuses/${{ needs.pr-info.outputs.sha }}'
+          context: ${{ env.STATUS_NAME }}
+          state: pending
+          description: 'Fedora 38 - Runner has been deploying...'
+          target_url: '${{ steps.job-log-url.outputs.html_url }}'
+        env:
+          GITHUB_TOKEN: ${{ secrets.PAT }}
+
+  iot-simplified-f38:
+    needs: [pr-info, pre-iot-simplified-f38]
+    runs-on: [kite, x86_64, gcp, fedora-38, medium]
+    timeout-minutes: 100
+    env:
+      STATUS_NAME: Tests / iot-simplified-f38
+
+    steps:
+      - name: Get Current Job Log URL
+        id: job-log-url
+        run: |
+          sudo dnf install -y jq
+          JOB_HTML_URL=$(curl --get -Ss -H "Authorization: token ${GITHUB_TOKEN}" -H "Accept: application/vnd.github.v3+json" "https://api.github.com/repos/osbuild/rhel-edge-ci/actions/runs/${GITHUB_RUN_ID}/jobs?per_page=30" | jq -r --arg job_name "iot-simplified-f38" '.jobs | map(select(.name == "iot-simplified-f38")) | .[0].html_url')
+          echo "html_url=$JOB_HTML_URL" >> $GITHUB_OUTPUT
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Create in-progress status
+        uses: octokit/request-action@v2.x
+        with:
+          route: 'POST /repos/osbuild/osbuild-composer/statuses/${{ needs.pr-info.outputs.sha }}'
+          context: ${{ env.STATUS_NAME }}
+          state: pending
+          description: 'Fedora 38 - Test has been running...'
+          target_url: ${{ steps.job-log-url.outputs.html_url }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.PAT }}
+
+      - name: Clone repository
+        uses: actions/checkout@v3
+        with:
+          repository: ${{ needs.pr-info.outputs.sender_repo }}
+          ref: ${{ needs.pr-info.outputs.sender_branch }}
+          fetch-depth: 0
+
+      - name: Make a deployment
+        run: schutzbot/deploy.sh
+        env:
+          GIT_COMMIT: ${{ needs.pr-info.outputs.sha }}
+        timeout-minutes: 20
+
+      - name: run ostree-simplified-installer.sh
+        run: /usr/libexec/tests/osbuild-composer/ostree-simplified-installer.sh
+        env:
+          DOWNLOAD_NODE: ${{ secrets.DOWNLOAD_NODE }}
+        timeout-minutes: 60
```

Comment:
60 minutes is not enough for simplified installer test.

Reply:
fixed, change to 150 as same as other simplified test cases.